appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/campaign/CampaignMapper.java

```diff
@@ -53,5 +56,20 @@ private static String GetStringValue(String paramName, String response) {
 
     return "";
   }
+
+  public static Campaign mapCampaignFromBundle(Bundle response) {
+
+    try {
+      int responseCode = response.getInt("RESPONSE_CODE");
+      if (responseCode == ResponseCode.OK.getValue()) {
+        return new Campaign(new BigInteger(response.getString("CAMPAIGN_ID")), "");
+      }
+    } catch (Exception e) {
+      e.printStackTrace();;
+    }
+
+    Log.d(CampaignMapper.class.getName(), "No campaign is available.");
+    return new Campaign(new BigInteger("-1"), "");
```

Comment:
The definition of an invalid campaign id should be obtained from the Campaign class itself and not defined hardcoded. Please use INVALID_CAMPAIGN field from Campaign class.

The defined log may come to be redundant since we use the same log in the manager that processes the campaign id:
`if (!campaign.hasCampaign()) {
      Log.d(TAG, "No campaign is available.");
      stopProcess();
    }`